api/HTMLAreaElement.json

```diff
@@ -239,54 +239,6 @@
           }
         }
       },
-      "media": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLAreaElement/media",
```

Comment:
404 and I've cleaned up https://wiki.developer.mozilla.org/en-US/docs/Web/API/HTMLAreaElement